setup.py

```diff
@@ -23,11 +28,29 @@
 assert_supported_version()
 
 
+def create_commit_file():
+    """
+    Creates COMMIT file with `HEAD` sha if `COALIB_DIR` is a git repo.
+    """
+    current_commit = get_current_commit(COALIB_DIR)
+    if(current_commit != ''):
```

Comment:
```diff
-if(current_commit != '')
+if (current_commit != '')
```

but ... wtf...

If you can not get a commit identifier, it is probably because the user is installing from a tarball.  You must write a COMMIT file so that the package doesnt try to run `git` at startup every time the package is loaded.